keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterSpec.kt

```diff
@@ -94,7 +94,7 @@ data class TitusClusterSpec(
 }
 
 data class TitusServerGroupSpec(
-  val container: Container,
+  val container: Container? = null,
```

Comment:
Container can't be null, ever. Why the change?

Reply:
I think so the class can be used for overrides, in which case all properties need to be nullable?

Reply:
👍 alright